integration_test_lite.py

```diff
@@ -116,25 +116,28 @@ async def bad_response(self, endpoint):
                 ) as response:
 
                     response_code = response.status
-                    if response_code != 200:
-                        api_info['response_code'] = response_code
+                    api_info['response_code'] = response_code
 
+                    # only add response body if test was failure
+                    if response_code != 200:
                         try:
                             api_info['response_body'] = await response.json()
                         except Exception as error:
                             api_info['response_body'] = {
                                 'raw': await response.text(),
                                 'error': str(error)
                             }
-                        return api_info
-                    query_param_string = urlencode(query_params)
-                    print(f'    [{response_code}] {url}?{query_param_string}')
+                    else:
+                        query_param_string = urlencode(query_params)
+                        print(f'    [{response_code}] {url}?{query_param_string}')
```

Comment:
A better way to add indentation is to use [`textwrap.indent`](https://docs.python.org/3.7/library/textwrap.html#textwrap.indent) somehting like:

```python
indent(text, ' ' * 4)
```